tailcfg/tailcfg.go

```diff
@@ -170,7 +170,8 @@ type CapabilityVersion int
 //   - 123: 2025-07-28: fix deadlock regression from cryptokey routing change (issue #16651)
 //   - 124: 2025-08-08: removed NodeAttrDisableMagicSockCryptoRouting support, crypto routing is now mandatory
 //   - 125: 2025-08-11: dnstype.Resolver adds UseWithExitNode field.
-const CurrentCapabilityVersion CapabilityVersion = 125
+//   - 126: 2025-08-21: add tailnet display name field to MapResponse
+const CurrentCapabilityVersion CapabilityVersion = 126
```

Comment:
this is harmless, but also unnecessary probably--- we only bump this when the server needs to know how to differentiate client capabilities. Without this PR, we could push down a display name to clients. And with this PR, we could also could. There's no difference from the server's point of view.